src/utils/popups.js

```diff
@@ -207,6 +215,24 @@ const sitePropertiesToHTML = ({ properties, source }) => {
   return sitePopupHTML("Monitoring Site", displayProps, url);
 };
 
+const bristolSitePropertiesToHTML = ({ properties }) => {
+  const latestRecordDateTime = new Date(properties.record.datetime);
+
+  const displayProps = {
+    Name: properties.label,
+    "Site Id": properties.siteId,
+    "Latest record datetime": `${latestRecordDateTime.toLocaleDateString()} ${latestRecordDateTime.toLocaleTimeString()}`,
```

Comment:
Bit wordy but the comma was being annoying lol
<img height=300 src="https://user-images.githubusercontent.com/32230328/203312577-0ac324ca-fe17-475e-8658-7ee5188ab1e1.png"></img><img height=300 src="https://user-images.githubusercontent.com/32230328/203312477-077a37b8-0c8d-4248-b631-62e027b28156.png"></img>